packages/flutter/test/rendering/view_test.dart

```diff
@@ -2,12 +2,23 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:ui' as ui;
+
 import 'package:flutter/rendering.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 import 'rendering_tester.dart';
 
 void main() {
+  // Create non-const instances, otherwise tests pass even if the
```

Comment:
add a test case for this to encode that two ViewConfiguration returned by this are not `identical`? (could be part of the `ViewConfiguration == and hashCode` test)

Reply:
This is also indirectly covered in the replace root layer test, but added a more explicit one to the == and hashcode test.